service/src/test/java/bio/terra/cbas/runsets/inputs/TestInputGeneratorBuildInputs.java

```diff
@@ -17,68 +21,68 @@ class TestInputGeneratorBuildInputs {
       new ObjectMapper().configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
 
   @Test
-  void stringLiteral() throws JsonProcessingException {
+  void stringLiteral() throws Exception {
```

Comment:
It's a test, so this isn't the end of the world, but if we **_could_** be more specific about what type(s) of exception this might throw, that would still be nice, IMO

Reply:
I agree. When the exception thrown is `JsonProcessingException` I get an error `Unhandled exception: bio.terra.cbas.common.exceptions.WorkflowAttributesNotFoundException`. Which is why I then changed it to the more general `Exception`. Is there a way to incorporate both?

Reply:
a quick search online suggests you could do something like:

```
void stringLiteral() throws JsonProcessingException, WorkflowAttributesNotFoundException {
    ...
}
```

https://stackoverflow.com/questions/2912565/throwing-multiple-exceptions-in-java

Reply:
I did this as well, but that caused errors in github tests, causing all of these to fail. Maybe I can add something like an `assert` to the end of the test to work around this?

Reply:
So
1. This was a low priority comment from me, feel free to ignore it if it's the only thing stopping you from merging.
2. It might be a useful learning experience to work out what the problem is, since exception wrangling is (unfortunately) quite a large part of Java programming... so we can mob if you're curious, or you can ignore this if you just want it to be done (and we can always revisit it later!)